src/components/settings/settings-access-status-types/settings-access-status-types.js

```diff
@@ -129,35 +175,53 @@ const SettingsAccessStatusTypes = ({
             formatter={formatter}
             label={intl.formatMessage({ id: 'ui-eholdings.settings.accessStatusTypes' })}
             onCreate={onCreateItem}
-            onDelete={onDelete}
+            onDelete={showConfirmDialog}
             onUpdate={onUpdate}
             readOnlyFields={['lastUpdated', 'records']}
             visibleFields={['name', 'description', 'lastUpdated', 'records']}
           />
         )}
       </IntlConsumer>
+      <ConfirmationModal
+        id="delete-access-status-type-confirmation-modal"
+        heading={<FormattedMessage id="ui-eholdings.settings.accessStatusTypes.delete" />}
+        message={<FormattedMessage id="ui-eholdings.settings.accessStatusTypes.delete.description" values={{ name: selectedStatusType?.attributes?.name || '' }} />}
```

Comment:
Please, move each prop to separate line
```suggestion
        message={
          <FormattedMessage
            id="ui-eholdings.settings.accessStatusTypes.delete.description"
            values={{ name: selectedStatusType?.attributes?.name || '' }}
          />
        }
```